less/home-page.less

```diff
@@ -1,6 +1,13 @@
 .home {
 
   .header-section {
+    img {
+      margin-bottom: 2rem;
+    }
+    p {
+      padding-bottom: 2rem;
+      border-bottom: #C0C0C0 dashed 2px;
```

Comment:
Didn't know you can do the values of border in reverse order. 😲 but I think it's best to get used to typing it in proper order. 😅